setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="edd-utils",
-    version="0.0.12",
+    version="0.0.13",
     author="Zak Costello, William Morrell, Mark Forrer, Tijana Radivojevic, Reinhard Gentz",
     author_email="tradivojevic@lbl.gov",
```

Comment:
How about having your email here instead?

Reply:
Good point, I'll switch it over to the development mailing list.